volumes.c

```diff
@@ -313,6 +313,7 @@ int pv_volume_mount(struct pv_volume *v)
 	int loop_fd = -1, file_fd = -1;
 	struct pantavisor *pv = pv_get_instance();
 	struct pv_state *s = pv->state;
+	struct pv_disk *d = v->disk;
```

Comment:
Why these changes in volumes.c? I think this is something we changed after creating this PR, so maybe it might bring back the several volumes per disk crash https://github.com/pantavisor/pantavisor/pull/205